src/containers/cooperation-details/add-course-modal-modal/AddCourseTemplateModal.styles.ts

```diff
@@ -4,10 +4,12 @@ import { TypographyVariantEnum } from '~/types'
 export const styles = {
   root: {
     width: { lg: '1130px', md: '786px', xs: '100%' },
-    height: '600px',
+    height: { xs: '100vh', sm: '600px' },
```

Comment:
I suggest using `svh` or `dvh` for mobile devices. [Here](https://dev.to/frehner/css-vh-dvh-lvh-svh-and-vw-units-27k4) you can find out how this approach can be useful.

Reply:
Thanks, replaced `vh` by `svh`